TGC/events/Filibuster.txt

```diff
@@ -0,0 +1,593 @@
+### Filabusters ###
+
+	## William Walker invades Nicaragua
+		country_event = {
+			id = 8870780
+			title = "EVTNAME8870780"
+			desc = "EVTDESC8870780"
+			picture = "Cubanrebels"
+
+			fire_only_once = yes
+			trigger = {
+				tag = NIC
+				year = 1854
+				NOT = { year = 1860 }
+				owns = 2198
+				owns = 2197
+				owns = 2199
+			}
+
+			mean_time_to_happen = {
+				days = 286
+			}
+
+			# Don't play as FIL
+			option = {
+				name = "EVTOPTA8870780"
+
+				prestige = -5
+				NIC = {
+					all_core = {
+						add_core = FIL
+					}
+					random_owned = {
+						limit = { province_id = 2198 }
+						secede_province = FIL
+					}
+					random_owned = {
+						limit = { province_id = 2197 }
+						secede_province = FIL
+					}
+				}
+
+				FIL = {
+					activate_technology = military_staff_system
+					activate_technology = iron_muzzle_loaded_artillery
+
+				}
+
+				USA = {
+					random_pop = {
+						limit = {
+							culture = dixie
+							is_primary_culture = no # Workaround to select Dixie only since has_pop_culture=dixie and culture=dixie both can select Yankee
+							type = soldiers
+						}
+						move_pop = 2198
+					}
+					random_pop = {
+						limit = {
+							culture = dixie
+							is_primary_culture = no
+							type = officers
+						}
+						move_pop = 2198
+					}
+				}
+
+				FIL = {
+					define_general = {
+						name = "William Walker"
+						personality = adventurer
+						background = relentless
+					}
+					money = 10000
+					firearms = 100
+				}
+
+				war = {
+					target = FIL
+					attacker_goal = { casus_belli = civil_war country = NIC }
+					defender_goal = { casus_belli = civil_war country = FIL }
+					call_ally = no
+				}
+
+				ai_chance = { factor = 1 }
+			}
+
+			# Play as FIL
+			option = {
+				name = "EVTOPTB8870780"
+
+				prestige = -5
+				NIC = {
+					all_core = {
+						add_core = FIL
+					}
+					random_owned = {
+						limit = { province_id = 2198 }
+						secede_province = FIL
+					}
+					random_owned = {
+						limit = { province_id = 2197 }
+						secede_province = FIL
+					}
+				}
+
+				FIL = {
+					activate_technology = military_staff_system
```

Comment:
This can be done in the country history file. It's not necessary to do it here, unless you want it to be conditional (i.e. depending on year, USA tech, NIC tech etc).

Reply:
Found better way to do this.